### PR TITLE
Add easy deploy button

### PR DESCRIPTION
**Pull request by asuresh4**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "website": "https://signalfx.com/",
+  "repository": "https://github.com/signalfx/heroku-signalfx-collector",
+	"env": {
+		"SFX_TOKEN": {
+			"description": "SignalFx access token of the org to which data needs to be sent"
+		},
+		"SFX_INGEST_URL": {
+			"description": "Ingest URL to which data needs to be sent"
+		},
+		"SFX_REALM": {
```
Comment (keitwb) [inline app.json]: Yes require this.  We are moving away from defaulting to us0.

File: app.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "signalfx",
+    "splunk",
+    "metrics",
+    "observability"
+  ],
+  "website": "https://signalfx.com/",
+  "repository": "https://github.com/signalfx/heroku-signalfx-collector",
+	"env": {
+		"SFX_TOKEN": {
+			"description": "SignalFx access token of the org to which data needs to be sent"
```
Comment (keitwb) [inline app.json]: This should be required.

File: app.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		},
+		"SFX_INGEST_URL": {
+			"description": "Ingest URL to which data needs to be sent"
```
Comment (keitwb) [inline app.json]: This should not be required, although we should absolutely NOT have the buildpack defaulting to us0 as a realm. It should require one of realm or ingest_url (probably not enforceable with this app.json format).

File: app.json (hunk 2 above)
Comment (asuresh4) [reply app.json]: The default for config is always required. https://devcenter.heroku.com/articles/app-json-schema#env

File: app.json (hunk 3 above)
Comment (asuresh4) [reply app.json]: Yeah, even though the app requires only one of the two config options, doesn't look like it enforceable here. Thats's why made `SFX_INGEST_URL` required and `SFX_REALM` not required. I don't believe we default to any realm. The app requires one of the two variables

File: app.json (hunk 2 above)
Comment (keitwb) [reply app.json]: It does not say anything about default being required, it just says it has to be a string.

Comment (keitwb) [reply app.json]: Actually leave this not required with no default because you might not need a token if sending to a gateway.

**Pull request merged**